src/main/java/com/google/sps/model/CalendarClient.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.model;
+
+import com.google.api.services.calendar.model.Event;
+import java.io.IOException;
+import java.util.List;
+
+/** Interface to handle get requests to the Calendar API. */
+public interface CalendarClient {
+  /**
+   * Get the events in a user's calendar.
+   *
+   * @return the list of Event from the calendar
+   * @throws IOException thrown when an issue occurs
+   */
+  List<Event> getCalendarEvents() throws IOException;
```

Comment:
Fly-by comment - it would be good to pass in the calendar/list to retrieve events from, even if you always ask for "primary" when you call this method. We know that eventually we want to specify the calendar(s), so let's make the API support it.

It would also be more consistent with TaskClient and GmailClient.